client/src/pages/Balance.js

```diff
@@ -0,0 +1,100 @@
+import React, { useContext, useState } from "react";
+import userContext from "../userContext";
+import Background from "../elements/Background";
+import StyledPaper from "../elements/StyledPaper";
+import PageHeader from "../elements/PageHeader";
+import {
+  Button,
+  Card,
+  Container,
+  CssBaseline,
+  Divider,
+  Grid,
+  Typography,
+} from "@material-ui/core";
+
+const BalancePage = ({ state, dispatch }) => {
+  const user = useContext(userContext);
+  const [topUp, setTopUp] = useState(2);
+
+  const handleClick = ({ target, ...e }) => {
+    // e.prototype.preventDefault();
+    const { id } = target.closest("button");
+    if (id === "incr") return setTopUp(topUp + 1);
+    if (id === "decr" && topUp > 0) return setTopUp(topUp - 1);
+  };
+
+  const handleCheckout = (e) => {};
+
+  return (
+    <Background solid>
+      <Container component="main" maxWidth="xs">
+        <CssBaseline />
+        <StyledPaper mt={20}>
+          {/* Page Heading */}
+          <PageHeader>Your Balance:</PageHeader>
+          <Typography
+            component="p"
```

Comment:
I rarely change components in Typography - any reason why you opted to do that?

Reply:
I'm not familiar with MaterialUI, but it seems really easy to fall out of line with a11y best practice. Seemed safer to err on the side of being overly imperative.